packages/android_alarm_manager_plus/example/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 #Tue Oct 05 12:15:50 EEST 2021
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-all.zip
```

Comment:
Would *-bin.zip be sufficient?

Reply:
It is fine to do more commits as we are squashing them on merge.